src/darwin/Framework/CHIP/CHIPIMDispatch.mm

```diff
@@ -156,7 +156,7 @@ bool IsClusterDataVersionEqual(const ConcreteClusterPath & aConcreteClusterPath,
     const EmberAfAttributeMetadata * GetAttributeMetadata(const ConcreteAttributePath & aConcreteClusterPath)
     {
         // Note: This test does not make use of the real attribute metadata.
-        static EmberAfAttributeMetadata stub = { .defaultValue = EmberAfDefaultOrMinMaxAttributeValue(uint16_t(0)) };
+        static EmberAfAttributeMetadata stub = { .defaultValue = EmberAfDefaultOrMinMaxAttributeValue(uint32_t(0)) };
         return &stub;
     }
```

Comment:
This is a comment on the zap repo and not this file itself (I don't know how to add comment to git submodule)

There is a regression in zap
https://github.com/project-chip/zap/issues/510

I will see tomorrow if zap folks ahve a fix. If not I will ask them if I could I branch from what we currently use in matter and add the one thing we need for this into that branch and point matter repo to that branch (I doubt it but that could allow this to move forward while that issue gets resolved)

Reply:
Confirmed https://github.com/project-chip/zap/issues/510 is no longer an issue with latest zap repo I am pulling in